prefect_transform/tasks.py

```diff
@@ -72,44 +53,29 @@ def create_materialization(
     Returns:
         An `MqlQueryStatusResp` object if `run_async` is `True`.
         An `MqlMaterializeResp` object if `run_async` is `False`.
-    """
-    # Raise error if both api_key and api_key_env_var are missing
-    if not (api_key or api_key_env_var):
-        msg = "Both `api_key` and `api_key_env_var` are missing."
-        raise TransformConfigurationException(msg)
-
-    # Raise error if api_key is missing and env var is not found
-    if not api_key and api_key_env_var not in os.environ.keys():
-        msg = "`api_key` is missing and `api_key_env_var` not found in env vars."
-        raise TransformConfigurationException(msg)
 
-    # Raise error if both mql_server_url and mql_server_url_env_var are missing
-    if not (mql_server_url or mql_server_url_env_var):
-        msg = "Both `mql_server_url` and `mql_server_url_env_var` are missing."
-        raise TransformConfigurationException(msg)
+    Example:
+    ```python
+        from prefect import flow
```

Comment:
This example needs one un-indent
![image](https://user-images.githubusercontent.com/15331990/185211825-bafbad97-2d68-47f4-9d0c-835feda30e34.png)